### PR TITLE
DBZ-5390 Reorder relational_property definition to give priority to out_of_line_constraint

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-5390

### CODE REVIEW

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4 [hunk 1]
```diff
@@ -3230,7 +3230,8 @@ end_time_column
     ;
 
 column_definition
-    : column_name (datatype | type_name)
+    : out_of_line_ref_constraint | out_of_line_constraint
```
Comment (ani-sha) [inline debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4:3233]: Hi @Naros, I am not entirely sure if this fix would be the right to do. Although the change now recognizes both the constraints as `out_of_line_ref_constraint` it leads to some test failures. What's your opinion on this?

Comment (ani-sha) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4]: This works. Thanks.

**Comment by ani-sha** [comment]
Upstream PR - https://github.com/antlr/grammars-v4/pull/2714

**Comment by jpechane** [comment]
@Naros @ani-sha Applied, thanks

**Pull request merged**